src/test/java/org/springframework/data/redis/serializer/GenericJackson2JsonRedisSerializerUnitTests.java

```diff
@@ -420,7 +423,7 @@ void deserializesJavaTimeFrimBytes() {
 	}
 
 	@Test // GH-2601
-	public void internalObjectMapperCustomization() {
```

Comment:
I just removed according to sonarlint plugin. It looks like an unnecessary keyword.